framespacer/R/asdataframe.R

```diff
@@ -0,0 +1,12 @@
+#' FrameSpace DataFrame to R DataFrame
+#' @description Converts a FrameSpace DataFrame into an R DataFrame
+#' @param dataframe \strong{list} FrameSpace DataFrame
+#' @return R DataFrame
+#' @export
+as.dataframe <- function(dataframe){
```

Comment:
It wouldn't be overriding it. It would only use this method when you pass something with your custom class signature. 

This is actually a very common pattern in R. 

An example you have probably come across is the `plot` function.  The code that is run when you call that function depends on the object you pass to it.